library/src/test/java/com/datatorrent/lib/db/jdbc/JdbcIOAppTest.java

```diff
@@ -126,6 +126,7 @@ public void testApplication() throws Exception
       Thread.sleep(3000);
 
       Assert.assertEquals("Events in store", 10, getNumOfEventsInStore());
+      lc.shutdown();
```

Comment:
I think it would be better to place this before the assertion. Also above test is another example of of the anti-pattern where we don't check for presence of results while waiting for arbitrary amount of time.